pyblish_nuke/plugins/select_host.py

```diff
@@ -1,12 +0,0 @@
-import pyblish.api
-
-
-@pyblish.api.log
-class SelectHostVersion(pyblish.api.Selector):
```

Comment:
Ping @tokejepsen as he's the one who implemented this.

@tokejepsen are you ok removing it? What was the original reason?

- https://github.com/pyblish/pyblish-nuke/commit/8bd4c730f2e655714e1a35492242e55c1b157720